core/src/com/unciv/logic/map/mapgenerator/MapGenerator.kt

```diff
@@ -429,34 +482,45 @@ class MapGenerationRandomness{
     }
 
 
-    fun chooseSpreadOutLocations(number: Int, suitableTiles: List<TileInfo>, initialDistance: Int): ArrayList<TileInfo> {
+    fun chooseSpreadOutLocations(number: Int, suitableTiles: List<TileInfo>, mapRadius: Int): ArrayList<TileInfo> {
+        // Determine sensible initial distance from number of desired placements and mapRadius
+        // empiric formula comes very close to eliminating retries for distance
+        val sparsityFactor = (suitableTiles.size.toDouble() / HexMath.getNumberOfTilesInHexagon(mapRadius)).pow(0.4)
+        val initialDistance = max(1, (mapRadius * 1.2 / number.toDouble().pow(0.4) * sparsityFactor + 0.5).toInt())
```

Comment:
Shouldn't this be `min(1, [complicated formula])`? As we later loop `for (distance in initialDistance downTo 1` which is difficult if `initialDistance <= 1`.